pgcli/main.py

```diff
@@ -757,6 +759,48 @@ def get_last_query(self):
         """Get the last query executed or None."""
         return self.query_history[-1][0] if self.query_history else None
 
+    def format_output(self, title, cur, headers, status, settings):
```

Comment:
With the new change it looks like format_output doesn't need to be part of the PGCli class. The self variable is not used anywhere. 

We can either make it a @staticmethod if we think that's appropriate or we can move it outside of this class. What do you think?